measured-rails.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "measured", Measured::Rails::VERSION
 
-  spec.add_runtime_dependency "railties", ">= 4.2"
+  spec.add_runtime_dependency "railties", ">= 5.2"
   spec.add_runtime_dependency "activemodel", ">= 4.2"
```

Comment:
Should `activemodel` be locked to `>= 5.2` too?

Reply:
Good call, yes. 

I'll bump this then merge, thanks.